webcomponents/tool/src/main/frontend/js/sakai-date-fns.js

```diff
@@ -0,0 +1,24 @@
+import { formatDistance } from "./assets/date-fns/esm/index.js";
+import { ar, bg, ca, enGB, enUS, es, eu, ro, tr } from "./assets/date-fns/esm/locale/index.js";
+import { getUserLocale } from "./sakai-portal-utils.js";
+
+const locales = { ar, bg, ca, enGB, enUS, es, eu, ro, tr };
+
+export const sakaiFormatDistance = (to, from) => {
+
+  const options = { addSuffix: true, lang: getUserLocale() };
+
+  if (options.lang) {
+    if (options.lang.includes("-")) {
+      const parts = options.lang.split("-");
+      // Try the langcountry key, but fallback to just the language if that doesn't exist
+      options.locale = locales[parts.join("")] || locales[parts[0]];
+    } else {
+      options.locale = locales[options.lang];
+    }
+  } else {
+    options.locale = locales[enUS];
```

Comment:
Can you move to the default to the top instead? 

options.locale = locales[enUS];
if (options.lang) {

What happens if both of these are null?

locales[parts.join("")] || locales[parts[0]];

Reply:
can you move the default up and out of the else statement?